dae/setup.py

```diff
@@ -96,6 +96,7 @@
     stats_liftover=dae.tools.stats_liftover:main
 
     import_tools=dae.import_tools.cli:main
+    task_graph_demo=dae.task_graph.demo_graphs_cli:main
```

Comment:
Do we really want that? Do we really want people when they install gpf to get a `task_graph_demo` executable on their path? A demo that doesn't demo anything client related.

Reply:
fixed